apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -147,6 +147,20 @@ type Application struct {
 	//
 	// +optional
 	Selector *LabelSelector `json:"selector,omitempty"`
+
+	// Group limit applied to the aggregated log
+	// flow to this input. The total log flow from this input
+	// cannot exceed the limit.
+	//
+	// +optional
+	GroupLimit *LimitSpec `json:"groupLimit,omitempty"`
```

Comment:
@Pranjal-Gupta2 @alanconway why do these only apply to application logs?  Why can they not apply to infrastructure container logs? or audit logs? or journal logs?  I believe especially infra journal logs, maybe  container logs are chatty

Reply:
@jcantrill  For the first version, we just want to add flow control policies to the Application Logs.

Reply:
+1